docs/1.getting-started/6.server.md

```diff
@@ -0,0 +1,92 @@
+::
+
+## Powered by Nitro
+
+Nuxt 3 uses [`Nitro`](https://github.com/unjs/nitro), a project from [`UnJS`](https://unjs.io), for all of his server-side stuff, along to [`Vite`](https://vitejs.dev) or [`webpack`](https://webpack.js.org) for the client-side.
+
+::alert{type="info"}
+Note that [`Nitro`](https://github.com/unjs/nitro) is fully framework agnostic and can be used in any project, even in standalone to create a server.
+::
+
+Using [`Nitro`](https://github.com/unjs/nitro) give to Nuxt superpowers:
+
+- Full control on the server-side part of your app
+- Deploy your app anywhere
+- Hybrid mode for rendering your app
+
+## Full control on the server-side part of your app
+
+With [`Nitro`](https://github.com/unjs/nitro), you can easily manager the server-side of your Nuxt 3 app from endpoints to middlewares.
+
+Both endpoints and middlewares a simple as:
+
+```ts [~/server/{routes,middleware}/index.ts]
+export default defineEventHandler({
+  // ... Do whatever you want here
+})
+```
+
+And you can simply return `text`, `json`, `html` or even a `stream`.
+
+Out-of-the-box, [`Nitro`](https://github.com/unjs/nitro) supports **hot module reloading** and **auto-import** like the other parts of your Nuxt 3 app.
+
+::alert{type="info"}
+Learn more about endpoints and middlewares in [the related section](/docs/guide/directory-structure/server).
+::
+
+## Deploy your app anywhere
+
+[`Nitro`](https://github.com/unjs/nitro) offer the ability to deploy your Nuxt 3 anywhere, from a bare metal server to the edge network, with a start time near to 1ms. That's fast!
+
+Today, [`Nitro`](https://github.com/unjs/nitro) offers more than 15 presets to build your Nuxt 3 app, for cloud provider, including:
+
+- [Cloudflare Workers](https://workers.cloudflare.com/)
+- [Netlify Functions](https://www.netlify.com/products/functions/)
+- [Vercel Edge Network](https://vercel.com/docs/edge-network/introduction)
+- [AWS Lambda](https://aws.amazon.com/lambda/)
+- [IIS](https://www.iis.net/)
+
+Or for other runtime:
+
+- [Deno](https://deno.land/)
+- [Bun](https://bun.sh/)
+
+::alert{type="info"}
+Learn more about deployment in [the related section](https://nuxt.com/docs/getting-started/deployment).
+::
+
+## Hybrid mode for rendering your app
+
+Need for both static and dynamic pages in your Nuxt 3 app? [`Nitro`](https://github.com) covers your back!
+
+Nitro has a powerful feature called `routeRules` which allow you to define a set of rules to customize the rendering mode of all your routes, client and server from your Nuxt 3 app.
+
+```ts [~/nuxt.config.ts]
+export default defineNuxtConfig({
+  routeRules: {
+    '/': { prerender: true }, // Generated at build time for SEO purpose
+    '/api/*': { cache: { maxAge: 60 * 60 } } // Cached for 1 hour
+    '/old-page': { redirect: { to: { '/new-page', status: 302 } } // Redirection to avoid 404
+    // ...
+  }
+})
+```
+
```

Comment:
Maybe we can document the `defineRouteRules` composable there too.

Reply:
Oh! I wasn't aware of this composable! 😄

Reply:
There is no composable with that name in the docs. Will check code source and add documentation about it!

Reply:
It is still experimental, we should update the docs when not experimental anymore